include/svgpp/traits/attribute_type.hpp

```diff
@@ -123,6 +123,8 @@ template<>              struct attribute_type<tag::element::glyphRef, tag::attri
 template<>              struct attribute_type<tag::element::glyphRef, tag::attribute::y           > { typedef tag::type::number type; };
 template<>              struct attribute_type<tag::element::glyphRef, tag::attribute::dx          > { typedef tag::type::number type; };
 template<>              struct attribute_type<tag::element::glyphRef, tag::attribute::dy          > { typedef tag::type::number type; };
+template<>              struct attribute_type<tag::element::fePointLight, tag::attribute::x       > { typedef tag::type::number type; };
+template<>              struct attribute_type<tag::element::fePointLight, tag::attribute::y       > { typedef tag::type::number type; };
 template<>              struct attribute_type<tag::element::fePointLight, tag::attribute::dx      > { typedef tag::type::number type; };
```

Comment:
Could you please also delete attribute_type<...tag::attribute::dx/dy> for both fePointLight and feSpotLight.